### PR TITLE
add skipna argument to plot_posterior

### PR DESCRIPTION
**Pull request by aloctavodia**
Plot posterior fails if `nan` are present, this adds a skipna argument, defaults to False. 

- [x] Follows [official](https://github.com/arviz-devs/arviz/blob/master/CONTRIBUTING.md#pull-request-checklist) PR format
- [x] New features are properly documented (with an example if appropriate)?
- [x] Includes new or updated tests to cover the new feature
- [x] Code style  correct (follows pylint and black guidelines)
- [x] Changes are listed in [changelog](https://github.com/arviz-devs/arviz/blob/master/CHANGELOG.md#v0xx-unreleased)

### CODE REVIEW

File: arviz/plots/plot_utils.py [hunk 1]
```diff
@@ -579,7 +585,10 @@ def calculate_point_estimate(point_estimate, values, bw="default", circular=Fals
         else:
             point_value = mode(values)[0][0]
     elif point_estimate == "median":
-        point_value = np.median(values)
+        if skipna:
```
Comment (canyon289) [inline arviz/plots/plot_utils.py:588]: We should add tests for this specific function as well at the plotting level

File: arviz/plots/plot_utils.py [hunk 2]
```diff
@@ -550,12 +550,15 @@ def calculate_point_estimate(point_estimate, values, bw="default", circular=Fals
...
         and a circular KDE is used. Only valid for 1D KDE. Defaults to False.
+    skipna=True,
+        If true ignores nan values when computing the hdi. Defaults to false.
 
     Returns
     -------
     point_value : float
         best estimate of data distribution
     """
+    print("ski", skipna)
```
[the remark this answers is not in the document]
Comment (aloctavodia) [reply arviz/plots/plot_utils.py:561]: My bad!

File: arviz/plots/plot_utils.py (hunk 1 above)
Comment (aloctavodia) [reply arviz/plots/plot_utils.py:588]: I have added a test for `skipna` (True or False) for both matplotlib and bokeh. What else is needed?

**Comment by aloctavodia** [comment]
@canyon289 I think this is ready to merge.

**Pull request merged**